README.md

```diff
@@ -1,161 +1,174 @@
-- [Main Concepts](https://docs.bentoml.org/en/latest/concepts/index.html) - A step-by-step tour for learning main concepts in BentoML
-- [Examples](https://github.com/bentoml/BentoML/tree/main/examples) - Gallery of sample projects using BentoML
-- [ML Framework Guides](https://docs.bentoml.org/en/latest/frameworks/index.html) - Best practices and example usages by the ML framework of your choice
-- [Advanced Guides](https://docs.bentoml.org/en/latest/guides/index.html) - Learn about BentoML's internals, architecture and advanced features
-- Need help? [Join BentoML Community Slack 💬](https://l.bentoml.com/join-slack)
+### 🍱 Bento is the container for AI apps
 
----
+* Open standard for building AI apps, pack your code, model files, dependencies, and runtime configurations in one [Bento “to-go”](https://docs.bentoml.com/en/latest/concepts/bento.html).
+* Auto-generate API servers, supporting REST API, gRPC, and long-running inference jobs.
+* Auto-generate Docker container images with just one command.
 
-## Highlights
+### 🚀 Deploy Anywhere
 
-🍭 Unified Model Serving API
+* One-click deployment to [☁️ BentoCloud](https://bentoml.com/cloud), the Serverless platform made for hosting AI apps.
+* Scalable BentoML deployment with [🦄️ Yatai](https://github.com/bentoml/yatai) on Kubernetes.
+* Deploy auto-generated container images anywhere docker runs.
 
-- Framework-agnostic model packaging for Tensorflow, PyTorch, XGBoost, Scikit-Learn, ONNX, and [many more](https://docs.bentoml.org/en/latest/frameworks/index.html)!
-- Write **custom Python code** alongside model inference for pre/post-processing and business logic
-- Apply the **same code** for online(REST API or gRPC), offline batch, and streaming inference
-- Simple abstractions for building **multi-model inference** pipelines or graphs
 
-🚂 **Standardized process** for a frictionless transition to production
+---
 
-- Build [Bento](https://docs.bentoml.org/en/latest/concepts/bento.html) as the standard deployable artifact for ML services
-- Automatically **generate docker images** with the desired dependencies
-- Easy CUDA setup for inference with GPU
-- Rich integration with the MLOps ecosystem, including Kubeflow, Airflow, MLFlow, Triton
+## Documentation
 
-🏹 **_Scalable_** with powerful performance optimizations
+* Installation: `pip install bentoml`
+* Full Documentation: [docs.bentoml.com](https://docs.bentoml.com/en/latest/)
+* Tutorial: [Intro to BentoML](https://docs.bentoml.com/en/latest/tutorial.html)
 
-- [Adaptive batching](https://docs.bentoml.org/en/latest/guides/batching.html) dynamically groups inference requests on server-side optimal performance
-- [Runner](https://docs.bentoml.org/en/latest/concepts/runner.html) abstraction scales model inference separately from your custom code
-- [Maximize your GPU](https://docs.bentoml.org/en/latest/guides/gpu.html) and multi-core CPU utilization with automatic provisioning
+### 🛠️ What you can build with BentoML
```

Comment:
Should we just link Gallery so we have one place to keep all deployment. We should move these descriptions to to gallery, too. I made a feature request to #design for adding gallery descriptions.